stdlib/map.gr

```diff
@@ -19,12 +23,31 @@ record Map<k, v> {
   mut buckets: Array<Option<Bucket<k, v>>>,
 }
 
+/**
+ * @section Values: Functions for working with the Map data type.
+ */
+
+/**
+ * Creates a new empty map with an initial storage of the given length. As values are added or removed, the length may grow or shrink. Generally, you won't need to care about the length of your map and can use `Map.make()` instead.
+ *
+ * @param storageLength: The initial storage length of the map
```

Comment:
```suggestion
 * @param size: The initial storage size of the map
```

Reply:
@spotandjake please make this change to Set too